tortoise/queryset.py

```diff
@@ -105,6 +105,7 @@ def __init__(self, model) -> None:
         self._annotations = {}  # type: Dict[str, Aggregate]
         self._having = {}  # type: Dict[str, Any]
         self._custom_filters = {}  # type: Dict[str, dict]
+        self._explain = False  # type: bool
```

Comment:
I don't want to add more parameters to QuerySet, as the copying of this is a known performance issue.
Rather make the `.explain()` method immediately execute the explanation?

Reply:
Yes, that makes sense even from a user perspective, I'll change that. :-)